mantidproject/mantid
# Fix issues from clang-tidy misc-*

Description of work.

This fixes a handful of issues found in the `clang-tidy` build on [Jenkins](http://builds.mantidproject.org/job/clang_tidy/).

**To test:**



* Code review should be sufficient.
* (optional) Verify that the number of clang-tidy warnings is reduced. 

There is no GitHub issue associated with this pull request.

**Release Notes** 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
